### DISCUSSION TITLE
How to stop showing deployment status on pull request when deploy with environment

### DISCUSSION THREAD

**Discussion by lemontea1011** [community/community#29234] [Pull Requests]
Is there any way to stop showing following status update on pull request?

![image](https://user-images.githubusercontent.com/51118016/184294916-a14ec4f3-27de-4bc8-91eb-30300606aafd.png)

In this example, I created an environment calls `my-test-env`, and I also defined a workflow running with the environment.
Everytime I raise a PR or push new commit into the branch in PR, it will run the workflow to do some jobs.

If there are too many environment I used in workflow, the pull request will become very messy, even hard to find the comments from other reviewers.

**Comment by WesleyYue** [comment]
Also have the exact same issue. Ended up using my ad blocker (ublock origin) with the following rule:

`github.com##div.js-timeline-progressive-focus-container.js-timeline-item > div.js-updatable-content.js-socket-channel`

**Reply by mmynk** [reply]
*filter

Nice! Sadly, it doesn't fix the hidden comments, still those annoying status are gone.

**Comment by mmynk** [comment]
This becomes super annoying with numerous CIs. If a PR has enough commits (or force pushes), the PR becomes difficult to manage.
At least PR comments should not be hidden due to these status, and subsequent ones could be squashed.

Or just give an option to hide these annoyances.
